blockonomics.php

```diff
@@ -200,9 +202,28 @@ public function hookPaymentOptions($params)
     public function getPaymentOption()
     {
         $offlineOption = new PaymentOption();
+        $active_cryptos = $this->getActiveCurrencies();
+        $logoHeight = Configuration::get('BLOCKONOMICS_LOGO_HEIGHT');
+
+        if (!isset($logoHeight) || !$logoHeight) {
```

Comment:
This can be removed as using the `|intval` filter in logo_height.tpl will equate BLOCKONOMICS_LOGO_HEIGHT to `0` if this is not set.

Reply:
Done